docs/adr/0015-external-facing-api-design-and-implementation.md

```diff
@@ -0,0 +1,68 @@
+# 15. External Facing API Design and Implementation
+
+Date: 2021-09-23
+
+## Status
+
+Proposed
+
+## Context
+
+The Office of Head Start is expanding data sharing across internal systems. To support this, TTA Hub would like
+to make documented APIs available to external systems in reliable, consistent, and secure ways.
+
+### Options Considered
+
+To achieve this, two paths were considered.
+
+#### Expose existing internal APIs
+
+TTA Hub has a robust set of APIs already implemented to support the React frontend.
+
+Pros:
+
+* already implemented
+* well documented
+
+Cons:
+
+* API is highly coupled to TTA Hub UI needs, will become more difficult to change rapidly
+* authentication is based on sessions stored in browser cookies
+
```

Comment:
Another con would be authorization is also based on the user list contained in the TTA hub, and managed by hub admins. Seems like a fair to assume not all users of the external API are going to be users in the TTA hub.